src/MahApps.Metro/Controls/Dialogs/LoginDialog.cs

```diff
@@ -206,6 +210,31 @@ public bool RememberCheckBoxChecked
             set => this.SetValue(RememberCheckBoxCheckedProperty, BooleanBoxes.Box(value));
         }
 
+        /// <summary>Identifies the <see cref="Icon"/> dependency property.</summary>
+        public static readonly DependencyProperty IconProperty
+            = DependencyProperty.Register(nameof(Icon),
+                                          typeof(object),
+                                          typeof(LoginDialog),
+                                          new PropertyMetadata());
+
+        public object? Icon
+        {
+            get => this.GetValue(IconProperty);
+            set => this.SetValue(IconProperty, value);
+        }
+
+        /// <summary>Identifies the <see cref="IconTemplate"/> dependency property.</summary>
+        public static readonly DependencyProperty IconTemplateProperty
+            = DependencyProperty.Register(nameof(IconTemplate),
+                                          typeof(DataTemplate),
+                                          typeof(LoginDialog));
+
+        public DataTemplate? IconTemplate
+        {
+            get => (DataTemplate?)this.GetValue(IconTemplateProperty);
+            set => this.SetValue(IconTemplateProperty, value);
+        }
```

Comment:
Is it not better to move this to the base class? Because it is used in each derived class. So a user can use this in a custom dialog too. WDYT?